src/appMain/android_platform_interface.cc

```diff
@@ -16,8 +16,8 @@ void StartSDL(JNIEnv* env, jobject);
 void StopSDL(JNIEnv* env, jobject);
 
 static JNINativeMethod s_methods[] = {
-   {"StartSDL", "()V", (void*)StartSDL},
-   {"StopSDL", "()V", (void*)StopSDL}
+   {"StartSDLNative", "()V", (void*)StartSDL},
+   {"StopSDLNative", "()V", (void*)StopSDL}
```

Comment:
@VadymLuchko I think you can rename corresponding methids too
```suggestion
   {"StartSDLNative", "()V", (void*)StartSDLNative},
   {"StopSDLNative", "()V", (void*)StopSDLNative}
```

Reply:
done in 780cf0e02b9d5e377008109f118e8866fada65f6